### ISSUE TITLE
Hackage?

### ISSUE THREAD

**Issue by vladfi1** [ajtulloch/caffegraph#8]
Is this intended to be installed via hackage? I do see that there is a similar "caffegraph" package there.

**Comment by ajtulloch** [comment]
Yeah, I uploaded an old version. I can upload a new one - thanks!
